src/utilities/new_versions.jl

```diff
@@ -166,24 +172,30 @@ function continue_with_pr(dep::DepInfo, bump_compat_containing_equality_specifie
     # Determine if we need to make a new PR
     if (!isnothing(dep.version_spec) && !isnothing(dep.latest_version)) &&
        dep.latest_version in dep.version_spec
+        # TODO: This seems too noisy, is there a better fix?
```

Comment:
Using LoggingExtras to filter out messages you don't want at the end user level.
Maybe add a `group` arg here so you can filter them more easily

Reply:
hmm interesting, I'll look into this thanks!

Reply:
I think I'll open an issue for this, and just let the output happen in this PR for now.